dbt/adapters/firebolt/connections.py

```diff
@@ -77,7 +77,7 @@ class FireboltConnectionManager(SQLConnectionManager):
     def open(cls, connection):
         if connection.state == 'open':
             return connection
-        credentials = cls.get_credentials(connection.credentials)
+        credentials = connection.credentials
```

Comment:
what's the reason for this change?

Reply:
Because `get_credentials()` is

```
def get_credentials(cls, credentials):
        return credentials
        ```
        
Seemed pointless to me. In fact, I ought to remove `get_credentials()`, as it's not in use anymore.